save-plugins/warn-plugin/src/commonMain/kotlin/org/cqfn/save/plugin/warn/WarnPlugin.kt

```diff
@@ -81,16 +95,35 @@ class WarnPlugin(testConfig: TestConfig) : Plugin(testConfig) {
         )
     }
 
+    /**
+     * @param path
+     * @return name of the temporary file
+     */
+    internal fun createTestFile(path: Path): String {
+        val tmpDir = (FileSystem.SYSTEM_TEMPORARY_DIRECTORY / WarnPlugin::class.simpleName!!)
+
+        if (fs.exists(tmpDir)) {
+            fs.deleteRecursively(tmpDir)
```

Comment:
This directory should be created per SAVE execution, because different threads running the same plugin will create files there. You could check if your counter is 0 and then delete, otherwise do nothing